sympy/core/assumptions.py

```diff
@@ -6,8 +6,9 @@
 
 Assumptions determine certain properties of symbolic objects. Assumptions
 can have 3 possible values: True, False, None.  None is returned when it is
-impossible to say something about the property. For example, a generic Symbol
-is not known beforehand to be positive.
+impossible to say something about the property.  For example, a generic Symbol
+is not known beforehand to be positive.  True, if the object has the property
+and False otherwise (i.e. also if the property doesn't make sense for the object).
```

Comment:
nit:

can have 3 possible values: True, False, None.  True is returned if the object has the
property (and False is returned if it doesn't). When the property cannot be determined
(or when a method is not implemented) None will be returned, e.g. a generic symbol, x,
may or may not be positive so a value of None is returned for x.is_positive.

OR

+impossible to say something about the property.  (For example, a generic Symbol
+is not known beforehand to be positive.)  False is used when the object either
+doesn't (or can't) have the property and True is returned when the object has the property.